packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -491,6 +495,39 @@ class ExpansionTile extends StatefulWidget {
   /// than supplying a controller.
   final ExpansionTileController? controller;
 
+  /// Whether this list tile is part of a vertically dense list.
+  ///
+  /// If this property is null then its value is based on [ListTileTheme.dense].
+  ///
+  /// Dense list tiles default to a smaller height.
+  ///
+  /// It is not recommended to set [dense] to true when [ThemeData.useMaterial3] is true.
+  final bool? dense;
+
+  /// Defines how compact the list tile's layout will be.
+  ///
+  /// {@macro flutter.material.themedata.visualDensity}
+  ///
+  /// See also:
```

Comment:
Not sure if this "See also" part is still necessary because we already use `{@macro flutter.material.themedata.visualDensity}` above for `ThemeData.visualDensity`.

Reply:
That's right